.github/workflows/forge-unstable.yaml

```diff
@@ -145,120 +109,110 @@ jobs:
     needs: determine-test-metadata
     uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
     secrets: inherit
     with:
-      FORGE_NAMESPACE: forge-continuous-e2e-single-vfn
-      # Run for 8 minutes
-      FORGE_RUNNER_DURATION_SECS: 480
-      FORGE_TEST_SUITE: single_vfn_perf
+      FORGE_NAMESPACE: forge-twin-validator
+      FORGE_RUNNER_DURATION_SECS: 900
+      FORGE_TEST_SUITE: twin_validator_test
       POST_TO_SLACK: true
 
-  run-forge-state-sync-perf-fullnode-execute-test:
+  run-forge-validator-reboot-stress-test:
     if: ${{ github.event_name != 'pull_request' }}
     needs: determine-test-metadata
     uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
     secrets: inherit
     with:
-      FORGE_NAMESPACE: forge-state-sync-perf-fullnode-execute
+      FORGE_NAMESPACE: forge-validator-reboot-stress
       # Run for 40 minutes
       FORGE_RUNNER_DURATION_SECS: 2400
-      FORGE_TEST_SUITE: state_sync_perf_fullnodes_execute_transactions
+      FORGE_TEST_SUITE: validator_reboot_stress_test
       POST_TO_SLACK: true
 
-  run-forge-state-sync-perf-fullnode-fast-sync-test:
+  run-forge-three-region-different-node-speed:
     if: ${{ github.event_name != 'pull_request' }}
     needs: determine-test-metadata
     uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
     secrets: inherit
     with:
-      FORGE_NAMESPACE: forge-state-sync-perf-fullnode-fast-sync
-      # Run for 40 minutes
-      FORGE_RUNNER_DURATION_SECS: 2400
-      FORGE_TEST_SUITE: state_sync_perf_fullnodes_fast_sync
-      POST_TO_SLACK: true
+      IMAGE_TAG: ${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
+      FORGE_NAMESPACE: forge-three-region-with-different-node-speed
+      FORGE_RUNNER_DURATION_SECS: 3600
+      FORGE_TEST_SUITE: three_region_simulation_with_different_node_speed
+      FORGE_ENABLE_FAILPOINTS: true
+      POST_TO_SLACK: ${{ needs.determine-test-metadata.outputs.BRANCH == 'main' }} # only post to slack on main branch
 
-  run-forge-state-sync-perf-validator-test:
+  run-forge-changing-working-quorum-test-high-load:
     if: ${{ github.event_name != 'pull_request' }}
     needs: determine-test-metadata
     uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
     secrets: inherit
     with:
-      FORGE_NAMESPACE: forge-state-sync-perf-validator
-      # Run for 40 minutes
-      FORGE_RUNNER_DURATION_SECS: 2400
-      FORGE_TEST_SUITE: state_sync_perf_validators
-      POST_TO_SLACK: true
+      FORGE_NAMESPACE: forge-changing-working-quorum-test-high-load
+      FORGE_RUNNER_DURATION_SECS: 900
+      FORGE_TEST_SUITE: changing_working_quorum_test_high_load
+      POST_TO_SLACK: ${{ needs.determine-test-metadata.outputs.BRANCH == 'main' }} # only post to slack on main branch
+      FORGE_ENABLE_FAILPOINTS: true
 
-  run-forge-state-sync-slow-processing-catching-up-test:
+  run-forge-network-partition-test:
```

Comment:
not flaky any more, fixed on 2023-02-08T09:31:41Z, failed only once since then, on container creation:

https://cloud.us.humio.com/github/search?live=false&query=%23type%3Dgithub-event%20event_type%3Djob%7Cgithub.job.conclusion!%3Dcancelled%20github.job.conclusion!%3Dskipped%20github.job.conclusion!%3Dnull%20github.workflow.head_branch%3Dmain%20github.job.name%3D%2F.*forge%24%2F%7Cgithub.job.html_url%3D%2Faptos-core%5C%2Factions%2F%7Celapsed%20%3A%3D%20((((now())-(%40timestamp))%2F(24))%2F(3600))%2F(1000)%7Cmath%3Aceil(elapsed%2Cas%3Ddays_ago)%7Ccase%20%7B%20github.job.conclusion%3D%22success%22%7Csuccess%20%3A%3D%201%3B*%7Csuccess%20%3A%3D%200%7D%7Cfailure%20%3A%3D%20(1)-(success)%7Ccase%20%7B%20days_ago%3D1%7Csuccess_1d%20%3A%3D%20success%3B*%7Csuccess_1d%20%3A%3D%200%7D%7Ccase%20%7B%20days_ago%3D1%7Ccount_1d%20%3A%3D%201%3B*%7Ccount_1d%20%3A%3D%200%7D%7Ccase%20%7B%20days_ago%3C%3D3%7Csuccess_3d%20%3A%3D%20success%3B*%7Csuccess_3d%20%3A%3D%200%7D%7Ccase%20%7B%20days_ago%3C%3D3%7Ccount_3d%20%3A%3D%201%3B*%7Ccount_3d%20%3A%3D%200%7D%7Ccase%20%7B%20days_ago%3E3%7Csuccess_before_3d%20%3A%3D%20success%3B*%7Csuccess_before_3d%20%3A%3D%200%7D%7Ccase%20%7B%20days_ago%3E3%7Ccount_before_3d%20%3A%3D%201%3B*%7Ccount_before_3d%20%3A%3D%200%7D%20%7C%20github.job.name%3D%22run-forge-network-partition-test%20%2F%20forge%22%20%7Ctable(%5Bgithub.job.name%2C%20github.job.completed_at%2C%20github.job.conclusion%2C%20github.job.html_url%5D)%0A&start=30d&tz=America%2FLos_Angeles

https://github.com/aptos-labs/aptos-core/actions/runs/4172126361/jobs/7222853493